src/app/settings/components/settings/settings-menu/settings-menu.component.scss

```diff
@@ -18,10 +17,36 @@
       font-size: medium;
       display: block;
       text-decoration: none;
-      color: var(--primary-color);
+      color: var(--black-color);
       &.active-tab {
+        color: var(--white-color);
         background-color: var(--background-color);
```

Comment:
This color is way too dark for the selected settings menu item. You should look at the mockup for how it should look